docs/getting-started.md

```diff
@@ -5,79 +5,27 @@ layout: Doc
 
 # Setting Up Serverless Framework With AWS
 
-Getting started with Serverless Framework’s open-source CLI and AWS only takes a few minutes.
+Get started with Serverless Framework’s open-source CLI and AWS in minutes.
 
 ## Installation
 
-Install the serverless CLI via NPM:
+Install the `serverless` CLI via NPM:
 
 ```bash
 npm install -g serverless
 ```
 
-_Note: If you don’t already have Node on your machine, [install it first](https://nodejs.org/). We suggest using the latest LTS version of NodeJS._
+Note: If you don’t already have Node on your machine, [install it first](https://nodejs.org/). If you don't want to install Node or NPM, you can [install `serverless` as a standalone binary](./install-standalone.md).
 
-If you don't want to install Node or NPM, you can install a standalone binary.
+### Upgrade
 
-### Install as a standalone binary
+You can upgrade the CLI later by running the same command: `npm install -g serverless`.
 
-#### MacOS/Linux
-
-To install the latest version, run this command in your terminal:
-
-```bash
-curl -o- -L https://slss.io/install | bash
-```
-
-To install a specific version, you may set a VERSION variable, for example:
-
-```bash
-curl -o- -L https://slss.io/install | VERSION=2.72.2 bash
-```
-
-Then, open another terminal window to run the `serverless` program.
-
-#### Windows
-
-Install with [Chocolatey](https://chocolatey.org/):
-
-```bash
-choco install serverless
-```
-
-## Upgrade
-
-If `serverless` was installed via NPM, you can upgrade it via:
-
-```bash
-npm install -g serverless
-
-# You can also specify a major version:
-npm install -g serverless@2
-```
-
-If you installed `serverless` as a standalone binary, read the following section instead.
-
-### Standalone binary
-
-On MacOS/Linux, you can upgrade the standalone `serverless` binary by running:
-
-```bash
-serverless upgrade
-
-# You can also restrict the upgrade to the latest v2 version:
-curl -o- -L https://slss.io/install | VERSION=2.72.2 bash
-```
-
-On Windows, run:
-
-```bash
-choco upgrade serverless
-```
+To upgrade to a specific major version, specify it like this: `npm install -g serverless@2`. If you installed `serverless` as a standalone binary, [read this documentation instead](./install-standalone.md).
 
 ## Getting started
 
-Run the command below and follow the prompts:
+To create your first project, run the command below and follow the prompts:
```

Comment:
Goal: set expectations.

As a new visitor/user, my main question would have been: why should I run this command, and what will it do. "Getting started" is too vague.